src/Workspaces/Core/Portable/Classification/SyntaxClassification/AbstractNameSyntaxClassifier.cs

```diff
@@ -116,5 +117,37 @@ internal abstract class AbstractNameSyntaxClassifier : AbstractSyntaxClassifier
 
             result.Add(new ClassifiedSpan(span, ClassificationTypeNames.StaticSymbol));
         }
+
+        protected static bool IsStaticSymbol(ISymbol? symbol)
+        {
+            if (symbol is null || !symbol.IsStatic)
+            {
+                return false;
+            }
+
+            if (symbol.IsEnumMember())
+            {
+                // EnumMembers are not classified as static since there is no
+                // instance equivalent of the concept and they have their own
+                // classification type.
+                return false;
+            }
+
+            if (symbol.IsNamespace())
+            {
+                // Namespace names are not classified as static since there is no
+                // instance equivalent of the concept and they have their own
+                // classification type.
+                return false;
+            }
+
+            if (symbol.IsLocalFunction())
+            {
+                // Local function names are not classified as static since the
+                // the symbol returning true for IsStatic is an implementation detail.
+                return false;
+            }
+            return true;
```

Comment:
```suggestion

            return true;
```

nit: our codestyle is that there should generally be a blank line after a `}` and hte next statement.